pluspy.py

```diff
@@ -774,6 +774,30 @@ def parseError(a, r):
         shortest = r
     return (False, a, r)
 
+
+
+# Handy routine for rules that simply call other rules
+# It parses s using the given rule and returns a node (name, (t, a), r)
+# where name is the type of the AST node and (t, a) the result of
+# parsing given the rule.
+#
+# If t is a "Concat" rule (sequence of other rules), then you can
+# select a subset using the select argument
+#
+# Otherwise, if select is not None, the result of the rule is
+# directly returned without adding a new AST node
+def match(name, s, rule, select=None):
+    (t, a, r) = rule.parse(s)
```

Comment:
The parentheses around `t, a, r` can be removed.

Reply:
We could do that but that will go against style of the code in the file.